packages/core/src/modules/slack/use-cases/update-birthdates-to-db.ts

```diff
@@ -0,0 +1,30 @@
+import { db } from '@/infrastructure/database';
+import { slack } from '@/modules/slack/instances';
+
+export async function updateBirthdatesfromSlack() {
+  const members = await db
+    .selectFrom('students')
+    .select(['slackId'])
+    .where('slackId', 'is not', null)
+    .execute();
+
+  for (const member of members) {
+    await updateBirthdatebySlackId(member.slackId!);
+  }
+}
+
+async function updateBirthdatebySlackId(slackId: string) {
+  const user = await slack.users.profile.get({ user: slackId });
+
+  const birthdateFieldId = '[placeholder for the birthdate id]';
```

Comment:
I believe I'm not able to retrieve the exact birthdateFieldId as this is a custom field that was created by admins. As described in [Slack docs](https://api.slack.com/methods/users.profile.set#custom-profile), each field has a custom ID, and this ID is needed to access the birthdate value.

Reply:
Yeah the custom field would have to be provided by @ramiAbdou or someone with access to the Slack as an Admin, if im not mistaken.

Reply:
I updated this value in our production environment variables!